templates/pkg/resource/sdk.go.tpl

```diff
@@ -105,6 +105,13 @@ func (rm *resourceManager) sdkDelete(
 	r *resource,
 ) error {
 {{- if .CRD.Ops.Delete }}
+{{ $customMethod := .CRD.GetCustomImplementation .CRD.Ops.Delete }}
+{{ if $customMethod }}
```

Comment:
you can shorten the above to:

```
{{ if $customMethod := .CRD.GetCustomImplementation .CRD.Ops.Delete }}
```

like is done with line 32 in the sdk_find_read_one.go.tpl file below.